### ISSUE TITLE
Item frame furniture can be stacked in one place

### ISSUE THREAD

**Issue by Dogg123** [PluginBugs/Issues-ItemsAdder#3501]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### What happened?

I have searched everywhere for the solution, but couldn't find any. I want to make a vine block, but got stuck on an issue, where multiple fiurniture blocks can be placed on the same location

### Steps to reproduce the issue

1. Create an item frame furniture
2. Place it down
3 Place the furniture on the edge of the block

### Server version

1.19.4

### ItemsAdder Version

3.6.3-beta-13

### ProtocolLib Version

5.1.0

### LoneLibs Version

1.0.45

### Problematic items yml configuration file (optional)

```shell
hedera_placed:
    display_name: "Hedera"
    resource:
      material: PAPER
      generate: false
      model_path: block/hedera_placed
    behaviours:
      furniture:
        shift_interact_rotate: false
        entity: item_frame
        solid: false
        fixed_rotation: true
        cancel_drop: true
        placeable_on:
          floor: true
          ceiling: true
          walls: true
        sound:
          place:
            name: block.azalea.place
          break:
            name: block.azalea.break
    events:
      placed_itemframe:
        break:
          drop_item:
            item: hedera
  hedera:
    display_name: Wall hedera vines
    lore:
     - 'none'
    resource:
      material: PAPER
      generate: true
      textures:
      - item/hedera
    events:
      interact:
        right:
          place_furniture:
            furniture: hedera_placed
            decrement_amount: true
        right_shift:
          place_furniture:
            furniture: hedera_placed
            decrement_amount: true
```

### Screenshots/Videos (you can drag and drop files or paste links)

![place it down](https://github.com/PluginBugs/Issues-ItemsAdder/assets/92115584/b2391ac1-9b41-455c-a2ba-3b5205b72d60)
![hover on the edge of the block](https://github.com/PluginBugs/Issues-ItemsAdder/assets/92115584/838464bb-0366-47ce-8613-6e5455d33355)
![place](https://github.com/PluginBugs/Issues-ItemsAdder/assets/92115584/6e0691aa-8fbe-4533-8505-5f30a2688c12)

**Comment by Dogg123** [comment]
The same issue was mentioned in here:
https://github.com/PluginBugs/Issues-ItemsAdder/issues/3014
But I was still able to replicate the problem

**Comment by potapenchikov** [comment]
Yes, it's harder to do now, but it's still possible. I noticed this completely by accident and I tracked the position from the very edge of the block where the next face begins. I thought this could be fixed by increasing the hitbox by 0.1, but this does not solve the problem. It's pretty frustrating if players can use the entire furniture limit on one block and close another block on top, you'll just never know where all this furniture is

**Comment by potapenchikov** [comment]
It would also be good to prohibit putting any blocks on top of furniture using ITEM_FRAME
